newrelic/hooks/mlmodel_openai.py

```diff
@@ -484,25 +496,33 @@ async def wrap_embedding_acreate(wrapped, instance, args, kwargs):
     if not response:
         return response
 
-    response_model = response.get("model", "")
-    response_usage = response.get("usage", {})
     response_headers = getattr(response, "_nr_response_headers", None)
+
+    # In v1, response objects are pydantic models so this function call converts the object back to a dictionary for backwards compatibility
+    if OPENAI_V1:
+        response = response.model_dump()
```

Comment:
Just noticed this bug while running tests locally. We'll need to put this into a different variable as this currently changes the response of the wrapped method to `response.model_dump()`.